MasterSeed32.md

```diff
@@ -150,40 +176,43 @@ In order to recover a master seed, one needs a set of valid MS32 shares such tha
 If all the above conditions are satisfied, the `ms32_recover` function will return a MS32 secret when its argument is the list of MS32 shares with each share represented as a list of integers representing the characters converted using the bech32 character table from BIP-0173.
 
 ```python
-bech32_inv = [0, 1, 20, 24, 10, 8, 12, 29, 5, 11, 4, 9, 6, 28, 26, 31, 22, 18, 17, 23, 2, 25, 16, 19, 3, 21, 14, 30, 13, 7, 27, 15]
+bech32_inv = [
+    0, 1, 20, 24, 10, 8, 12, 29, 5, 11, 4, 9, 6, 28, 26, 31,
+    22, 18, 17, 23, 2, 25, 16, 19, 3, 21, 14, 30, 13, 7, 27, 15,
+]
 
 def bech32_mul(a, b):
-  res = 0
-  for i in range(5):
-    res ^= a if ((b >> i) & 1) else 0
-    a *= 2
-    if (32 <= a):
-      a ^= 41
-  return res
+    res = 0
+    for i in range(5):
+        res ^= a if ((b >> i) & 1) else 0
+        a *= 2
+        if (32 <= a):
```

Comment:
Do you want to make this

    a ^= 41 if (32 <= a) else 0

to match the style above?

Maybe for another PR?

Reply:
I think we should leave this as-is. I'm not sure if `<assignment> if <condition> else 0` is valid Python but I think it's surprising to have a side-effecting operation in an if expression.

Reply:
You are parsing it wrong it is `<value> if <condition> else 0` and we do it two lines above.

But I'm fine to consider it in a separate PR.

Reply:
Oh! I see. Yeah, I like that suggestion. But we'll do it in a separate PR.